src/app/api/life.graphql

```diff
@@ -0,0 +1,21 @@
+fragment LifeData on Life {
+    firstName
+    lastName
+    fullName
+    title
+    description
+    birthday
+    hobbies
+}
+
+query getLifeByID($id: String!) {
```

Comment:
Should also be an `ObjectId`

Reply:
Thank you! I have made the appropriate changes to the `id` type